website/blog/2024-07-08-building-your-semantic-layer-in-pieces.md

```diff
@@ -9,90 +9,176 @@ date: 2024-07-10
 
-We’re here to discuss another way: building a Semantic Layer in pieces. Our goal is to make sure you derive increased leverage and velocity from each step on your journey. If you’re eager to start building but have limited bandwidth (like most busy analytics engineers), this one is especially for you.
+<!--truncate-->
 
-## System of a noun: deciding what happens where
+## Fabric Best Practices
 
-When you’re using the dbt Semantic Layer, you want to _minimize_ _the modeling that exists outside of dbt_. Eliminate it completely if you can. Why?
+With the introduction of dbt Cloud support for Microsoft Fabric and Azure Synapse Analytics Dedicated SQL Pools, we're opening up new possibilities for analytics engineers in the Microsoft Ecosystem.
 
-- It’s **duplicative, patchy, and confusing** as discussed above.
-- It’s **less powerful**.
-- You **can’t** **test** it.
-- Depending on the tool, oftentimes you **can’t** **version control** it.
+The goal of this blog is to ensure a great experience for both
 
-What you want is a unified development flow that handles **normalized transformation in dbt models** and **dynamic denormalization in the dbt Semantic Layer** (meaning it dynamically combines and reshapes normalized data models into different formats whenever you need them).
+- end-user data analysts who rely upon the data products built with dbt and
+- the analytics engineers, who should predominately spend time creating and maintaining data products instead of maintaining and spinning up infrastructure
+- data engineers who focus on data movement and ingestion into Synapse
 
-:::info
-🏎️ **The Semantic Layer is a denormalization engine.** dbt transforms your data into clean, normalized marts. The dbt Semantic Layer is a denormalization engine that dynamically connects and molds these building blocks into the maximum amount of shapes available _dynamically_.
-:::
+To achieve this goal, this post will cover four main areas
+
+- Microsoft Fabric: the future of data warehousing in the Microsoft/Azure stack
+- strategic recommendations for provisioning Synapse environment
+- data modeling in dbt: Synapse style
+- Considerations for upstream and downstream of a Synapse-backed dbt project
+
+With that, let’s dive in!
+
+## Fabric is the future
+
+Many data teams currently use Azure Synapse dedicated pools. However, Fabric Synapse Data Warehouse is the future of data warehousing in the Microsoft Ecosystem.  Azure Synapse Analytics will remain available for a few more years, but Microsoft’s main focus is on Fabric as we can see in their roadmap and launches.
+
+Because data platform migrations are complex and time-consuming, it’s perfectly reasonable to still be using dbt with Azure Synapse for the next two years while the migration is under way. Thankfully, if your team already is using ASADSP, transitioning to the new Cloud offering will be much more straightforward than the migration from on-premise databases to the Cloud.
```

Comment:
curious, i wonder if there are more false positives when linting words that are acronyms. 

wonder if it would be better -- and if it's easy to do -- to just ignore acronyms.